person.rb

```diff
@@ -8,16 +14,18 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def of_age?
     @age >= 18
   end
 
-  public
-
   def can_use_services?
     of_age? || @parent_permission
   end
 
```

Comment:
- [ ] **A person has many rentals**: A method for adding rentals should be present
- Refer to my previous comment in book.rb